process/lifecycle.md

```diff
@@ -115,87 +82,94 @@ To be accepted at the Incubation stage, a project must meet the Sandbox requirem
+* A healthy number of public adopters are identified within the project ( using an ADOPTERS file or showcased on the project’s website ).
 * Have achieved and maintained an OpenSSF Best Practices Badge at the [gold level](https://bestpractices.coreinfrastructure.org/en/criteria/2).
 * Have a technical lead appointed for voting representation of the project to the TAC.
-* Be deemed by the TAC to add value to the mission of ASWF.
+* Be deemed by the TAC to add value to the mission of the Academy Software Foundation.
 * Obtain both a 2/3 supermajority vote of the TAC and an affirmative majority vote of the Governing Board
 
-*Since these metrics can vary significantly depending on the type, scope and size of a project, the TAC has final judgment over the level of activity that is adequate to meet these criteria. In general it will be necessary to (at least) demonstrate that the project is not overly reliant on any one individual or company, and can remain healthy in the event of a departure.
+*Since these metrics can vary significantly depending on a project's type, scope, and size, the TAC has final judgment over the level of activity adequate to meet these criteria. In general, it will be necessary to (at least) demonstrate that the project is not overly reliant on any one individual or organization and can remain healthy in the event of a departure.
 
 ### Benefits
 
 Adopted stage projects are eligible to receive the following benefits:
 
-* Be considered as one of the “TAC Projects” under the [ASWF Charter](https://github.com/AcademySoftwareFoundation/foundation/blob/master/CHARTER.md). 
- and may receive additional resources or support as determined by the Governing Board.
+* Be considered as one of the “TAC Projects” under the [Academy Software Foundation Charter](https://github.com/AcademySoftwareFoundation/foundation/blob/master/CHARTER.md) and may receive additional resources or support as determined by the Governing Board.
 * Have a voting representative on the TAC.
-* Right to refer to the project as an officially adopted project of ASWF, and receive highest priority for participating in events and other collaborative activities sponsored by ASWF.
+* Right to refer to the project as an officially adopted project of the Academy Software Foundation and receive the highest priority for participating in events and other collaborative activities sponsored by Academy Software Foundation.
 * A blog announcement or similar communication announcing the graduation of the project.
 
+### Expectations
+
+Adopted Stage projects will have an Annual Review to assess if it is still meeting those requirements. At the Annual Review, the TAC will vote to either: 
+
+* Renew the project at the Adopted Stage if it continues to [meet those requirements](#requirements-2).
+* Move the project to the [Archive Stage](#archive-stage). In this case, the trademark and any other assets will be returned to the project maintainers or an organization they designate.
+
```

Comment:
AFAIK we haven't done an explicit vote to renew the `Adopted Stage` status after a TAC presentation, should we have?

Reply:
It's a good idea; this gives the TAC a vehicle to move an Adopted Stage project to the Archived Stage if needed.

Reply:
"Adopted stage projects" ... "it is still meeting"

Plural vs singular?

Reply:
Fixed